src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -112,37 +131,39 @@ public void start(Stage primaryStage) throws Exception {
 
         // Main Page
 
-        HBox hboxClient = new HBox();
-        HBox hboxInput = new HBox();
-        VBox userVBox = new VBox();
-        VBox chatVBox = new VBox();
-        VBox convosVBox = new VBox();
-        BorderPane container = new BorderPane();
-        Button send = new Button("Send");
-        Button update = new Button("Update");
-        Button addConvo = new Button("Add Conversation");
-        Text userTitle = new Text("Users");
-        Text chatTitle = new Text("Conversation"); // changed based on name?
-        Text convosTitle = new Text("Conversations");
-        TextFlow userTf = new TextFlow(userTitle);
-        TextFlow chatTf = new TextFlow(chatTitle);
-        TextFlow convosTf = new TextFlow(convosTitle);
-        TextField input = new TextField();
-        // MenuBar menuBar = new MenuBar();
-        // Menu menuAdd = new Menu("Add Conversation");
-        // menuBar.getMenus().addAll(menuConvo);
-
-        ObservableList<String> usersList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+        hboxClient = new HBox();
+        hboxInput = new HBox();
+        userVBox = new VBox();
+        chatVBox = new VBox();
+        convosVBox = new VBox();
+        container = new BorderPane();
+        sendButton = new Button("Send");
+        updateButton = new Button("Update");
+        addConvoButton = new Button("Add Conversation");
+        userTitle = new Text("Users");
+        chatTitle = new Text("Conversation"); // changed based on name?
+        convosTitle = new Text("Conversations");
+        userTf = new TextFlow(userTitle);
+        chatTf = new TextFlow(chatTitle);
+        convosTf = new TextFlow(convosTitle);
+        input = new TextField();
+
+        // list of users
+        ObservableList<String> usersList = FXCollections.observableArrayList();
         ListView<String> users = new ListView<String>(usersList);
-        ObservableList<String> convoList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+
+        // list of conversations
+        ObservableList<String> convoList = FXCollections.observableArrayList();
         ListView<String> conversations = new ListView<String>(convoList);
-        ObservableList<String> messageList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+
+        // list of messages
+        ObservableList<String> messageList = FXCollections.observableArrayList();
         ListView<String> messages = new ListView<String>(messageList);
 
-        // userTf.setStyle("-fx-base: #b6e7c9;");
+        // add listener for when user presses send and add text to the messageList
+        sendButton.setOnAction(e -> messageList.addAll(input.getText()));
```

Comment:
Not yet because I wasn't sure how to do that until the conversation we had last night haha. The next step is to send the info to the server.